### DISCUSSION TITLE
are there esp32 CPU frequency/clock limits w LoRa radios?

### DISCUSSION THREAD

**Discussion by MrSniffer** [jgromes/RadioLib#758] [Q&A]
The esp32-s3 CPU can run at clock speeds from 10Mhz to 240Mhz. Are there concerns running at very low speed, e.g. 10Mhz, in order to save power? This may be more properly a question about the LoRa part than RadioLib. My specific concern relates to SX1265-based boards, such as the Heltec Wireless Stick V3.

**Comment by jgromes** [comment]
I don't think there should be any issues just because the CPU core clock is "slow" (is 10 MHz really slow?) - I've been using SX126x with ATmega328P running at 8 MHz. The default SPI speed is 2 MHz, so if the CPU clock is at least few times that then it should be OK. If you went below something like 4 MHz then I would consider lowering the SPI clock speed, but that's about it.

**Reply by MrSniffer** [reply]
Thanks for the encouraging response, Jan. I'll sleep better now.................
